src/Dumbo.php

```diff
@@ -133,7 +147,8 @@ public function handle(ServerRequestInterface $request): ResponseInterface
             $context = new Context(
                 $request,
                 $route ? $route["params"] : [],
-                $route ? $route["routePath"] : ""
+                $route ? $route["routePath"] : "",
+                $this->getConfig()
```

Comment:
I'm not sure about this, but I think we could just pass `$this`?

```suggestion
                $this
```

Reply:
Did you need to pass whole Class? In this case I think we just need to pass the value of the `configs` variable.